orthoinference/src/main/java/org/reactome/orthoinference/EWASInferrer.java

```diff
@@ -64,7 +64,7 @@
 						infReferenceGeneProductInst.addAttributeValue(species, speciesInst);
 						String referenceGeneProductSource = homologueSource.equals("ENSP") ? "ENSEMBL:" : "UniProt:";
 						infReferenceGeneProductInst.setAttributeValue(_displayName, referenceGeneProductSource + homologueId);
-						infReferenceGeneProductInst = InstanceUtilities.checkForIdenticalInstances(infReferenceGeneProductInst);
+						infReferenceGeneProductInst = InstanceUtilities.checkForIdenticalInstances(infReferenceGeneProductInst, null);
```

Comment:
It's a little unclear what the null parameter is here.  Maybe an overloaded `checkForIdenticalInstances` method that has a signature with just the first argument can be added in the InstanceUtilities class and it can call the existing method passing null as the second parameter.  That way you can pick the appropriate method signature depending on whether you need to pass one or two arguments.

Reply:
Yea that is a cleaner way of doing it. I'll make that change

Reply:
This I think will be added in its own PR in the future (added it to release notes)